lib/rules/method.js

```diff
@@ -35,19 +54,7 @@ module.exports = {
                 case "Identifier":
                 case "MemberExpression":
                     if ("property" in node.callee && node.arguments.length > 0) {
```

Comment:
This confuses me.
Do we even have `node.callee` for `identifier` and `MemberExpressions`? 🤔